sdk/lib/rtl/priv.c

```diff
@@ -486,3 +486,78 @@ RtlAdjustPrivilege(IN ULONG Privilege,
 
     return STATUS_SUCCESS;
 }
+
+/*
+ * @implemented
+ */
+NTSTATUS
+NTAPI
+RtlRemovePrivileges(IN HANDLE TokenHandle,
+                    IN PULONG PrivilegesToKeep,
+                    IN ULONG PrivilegeCount)
+{
+    DPRINT("RtlRemovePrivileges(%p, %p, %u)\n", TokenHandle, PrivilegesToKeep, PrivilegeCount);
+
+    /* Save privileges that should be keep */
+    UINT64 PrivilegesToKeepBitmap = 0;
+    if (PrivilegeCount)
+    {
+        ULONG ulCount;
+        PULONG pulPrivilegesToKeep = PrivilegesToKeep;
```

Comment:
If you used something like `PrivilegesToKeep[Ii]` you wouldn't need another pointer, you wouldn't need the `if (PrivilegeCount)` and the code would look less like straight out of IDA.